### PR TITLE
[Mailer][Webhook] Mailtrap webhook support

### PR DESCRIPTION
**Pull request by kbond**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Issues        | n/a
| License       | MIT

### CODE REVIEW

**Comment by fabpot** [comment]
@kbond https://github.com/symfony/symfony/pull/58248 has been merged now.

File: src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{
+    public function convert(array $payload): RemoteEvent
+    {
+        $type = match ($payload['event']) {
+            'delivery' => MailerDeliveryEvent::DELIVERED,
+            'open' => MailerEngagementEvent::OPEN,
+            'click' => MailerEngagementEvent::CLICK,
+            'unsubscribe' => MailerEngagementEvent::UNSUBSCRIBE,
+            'spam' => MailerEngagementEvent::SPAM,
+            'soft bounce', 'bounce' => MailerDeliveryEvent::BOUNCE,
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php:33]: is it really with a space in the middle? `soft bounce`?

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php (hunk 1 above)
Comment (kbond) [reply src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php:33]: Yep

Comment (OskarStark) [reply src/Symfony/Component/Mailer/Bridge/Mailtrap/RemoteEvent/MailtrapPayloadConverter.php:33]: thanks, weird 😄

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @kbond.

**Pull request merged**